### DISCUSSION TITLE
STAC & ESRI Arcgis Pro / Arcgis Catalogue

### DISCUSSION THREAD

**Discussion by WCMC-vblanque** [radiantearth/stac-spec#1288] [General STAC Q&A]
Hi everybody, 

I would like to export my ESRI Portal & Catalogue into a STAC Catalogue for all the Mapserver and Imageserver services registered in our portal. Is there any way to do this transformation (exportation) easily?

Currently I have only find the way to add a STAC connection in Arcgis Pro to pull data from external STAC.

It might be worth also to be sure our Mapserver and Imageserver are available and stored as COG in any AWS S3 repository (or similar)

best regards,

Vincent

**Comment by JC712DB** [comment]
Links to notebooks which connect to MPC stac

https://github.com/Esri/arcgis-for-mpc/tree/main/AMPC_Resources/Notebook_Examples
